ceph-container-flake8/build/build

```diff
@@ -21,7 +21,7 @@ function check(){
     while read -r filename; do
         pushd "$(dirname "$filename")"
         file=$(basename "$filename")
-        sudo docker run --rm -v "$(pwd)"/"$file":/"$file" eeacms/flake8 /"$file"
+        sudo docker run --rm -v "$(pwd)"/tox.ini:/tox.ini "$(pwd)"/"$file":/"$file" eeacms/flake8 /"$file"
```

Comment:
Not sure to understand why you expose tox.ini inside the container.
Also, you forgot to add another `-v`, one for each file/path you want to expose.

Reply:
Good catch. Thx !
I updated the commit message for a better explanation. Hope that's much better.